atlassian_backup-1.0.0/vendor/jira.py

```diff
@@ -0,0 +1,68 @@
+# Open Source code imported for use by author. Use make to update
```

Comment:
Similar question regarding license.

Reply:
Same answer as above. Atlassian Code is currently not licences so I assume it falls under Bitbucket Terms of Service.